test/normal/derived.gen.go

```diff
@@ -4142,7 +4142,7 @@ func deriveEqualPtrToName(this, that *Name) bool {
 func deriveEqualPtrToStructs(this, that *Structs) bool {
 	return (this == nil && that == nil) ||
 		this != nil && that != nil &&
-			this.Struct == that.Struct &&
```

Comment:
We are losing some testing of this case where structs can be compared with `==` instead of `Equal`, so maybe we should add another test case to generate where some of the fields do not have Equal methods, so this is still tested?

Reply:
Okay, I added a `StructWithoutMethod` to test.